### PR TITLE
refactor `toString` method in favor of polymorphism

### PR DESCRIPTION
**Pull request by vankop**
it allows easy way to implement `leadingComments` and `trailingComments` stringifying

### CODE REVIEW

File: src/selectors/node.js [hunk 1]
```diff
@@ -177,10 +177,14 @@ export default class Node {
         this.raws.spaces.after = raw;
     }
 
+    valueToString () {
+        return String(this.stringifyProperty("value"));
```
Comment (evilebottnawi) [inline src/selectors/node.js]: Move `String()` in  `stringifyProperty`

Comment (vankop) [reply src/selectors/node.js]: done

**Comment by vankop** [comment]
> Good job, can we add couple tests?

there are tests already, see https://github.com/postcss/postcss-selector-parser/blob/master/src/__tests__/util/helpers.js#L24

**Comment by evilebottnawi** [comment]
Need fix CI

**Comment by vankop** [comment]
Stringify is not stringify 🤔

**Comment by vankop** [comment]
@evilebottnawi problem with stringifying is here https://github.com/postcss/postcss-selector-parser/blob/master/src/selectors/namespace.js#L36

**Comment by evilebottnawi** [comment]
I don't know why previously developer use `get`/`set`, i think we should use `Node` from `postcss` and implement same interface

**Pull request merged**